tests/common.go

```diff
@@ -1228,6 +1234,36 @@ func ValidateMountOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*cha
 
 }
 
+// ValidateCreateOptionsWithPureVolumes is the ginkgo spec for executing file system options check for the given volume
+func ValidateCreateOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*chan error) {
```

Comment:
remove arg errChan if not using

Reply:
Now processError is handling arg errChan.